docs/http.md

```diff
@@ -9,16 +9,16 @@ Calling `robot.route('/my-app')` will return an [express](http://expressjs.com/)
 ```js
 module.exports = robot => {
   // Get an express router to expose new HTTP endpoints
-  const app = robot.route('/my-app');
+  const app = robot.route('/my-app')
 
   // Use any middleware
-  app.use(require('express').static(__dirname + '/public'));
+  app.use(require('express').static('public'))
```

Comment:
is there a reason this is getting removed? Is it necessary for express?

Reply:
Standard was warning not to use `+` for paths, but instead use `require('path').join(…)`. I removed this just to simplify. It's not necessary (it will be relative to `$PWD` without an absolute path).